tests/Hmac/HmacAuthenticatorTest.php

```diff
@@ -30,22 +31,10 @@ public function testConstructor(): void
         $this->assertInstanceOf(AuthenticatorInterface::class, $this->authenticator);
     }
 
-    /**
-     * @covers \Starlit\Request\Authenticator\Hmac\HmacAuthenticator::__construct()
-     */
-    public function testConstructorWithEmptyStringAsSecretThrowsException()
```

Comment:
Isn't this a quite nice fail safe to have in place?

Reply:
it is not needed anymore, as the secret gets only injected into the `HmacGenerator` and not the `HmacAuthenticator`. You can find the [test for the HmacGenerator here](https://github.com/starweb/starlit-request-authenticator/pull/4/files#diff-9759148343d07e6135732c93304967d9R52)

Reply:
Excellent! 👍